ml/src/test/scala/ai/h2o/sparkling/ml/features/H2OTargetEncoderTestSuite.scala

```diff
@@ -322,4 +322,130 @@ class H2OTargetEncoderTestSuite extends FunSuite with Matchers with SharedH2OTes
 
     TestFrameUtils.assertDataFramesAreIdentical(transformedByModel, transformedByMOJOModel)
   }
+
+  test("TargetEncoderModel returns the same result regardless the order of the inputCols specification") {
```

Comment:
Should we mark this test as skipped until it is fixed in H2O?

Reply:
Will do thanks!